src/renderer/page/settings/view.jsx

```diff
@@ -348,12 +348,17 @@ class SettingsPage extends React.PureComponent<Props, State> {
                 name="encrypt_wallet"
                 onChange={e => this.onChangeEncryptWallet(e)}
                 checked={walletEncrypted}
-                postfix={__('Encrypt my wallet with a custom password.')}
-                helper={__(
```

Comment:
We should still use this `helper` prop. You can just add the button at the end of it.